tests/expected/function-contract/history/ui/statement.expected

```diff
@@ -0,0 +1 @@
+error
```

Comment:
What is the error message that the user would see in this case?

Reply:
```
error[E0433]: failed to resolve: maybe a missing crate `core`?
 --> tests/expected/function-contract/history/ui/statement.rs:5:41
  |
5 | #[kani::ensures(|result| old(let x = 1+1;) old(*ptr + 1) == *ptr)]
  |                                         ^ maybe a missing crate `core`?
  ```
  Not sure if it's the error message we actually want, but it's also very malformated so it's really hard to get a good error message.

Reply:
@celinval should we create a GitHub issue to improve these error messages on a separate PR?

Reply:
Sure. I'm OK with this being a follow up PR, but we definitely need to improve these messages.

Reply:
Issue mentioned in #3273